hackathon_site/event/jinja2/event/landing.html

```diff
@@ -0,0 +1,306 @@
+                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
+                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
+                <br />
+            </div>
+            <p class="center">
+                If you would like to sponsor us, feel free to reach out:
+                <a href="mailto:hello@email.com" class="hoverLink">hello@email.com</a>
+            </p>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <div class="section">
+        <div class="s12 m8 flexColCenter">
+            <h3 class="center">Check out last year!</h3>
+
+            <video class="video center" controls>
+                <source src="https://file-examples-com.github.io/uploads/2017/04/file_example_MP4_1280_10MG.mp4" type="video/mp4" />
+                Your browser does not support the video tag.
+            </video>
+        </div>
+    </div>
+    <br /><br />
+</div>
+
+<div class="carousel">
+    <!-- IMAGES MUST ALL BE THE SAME SIZE -->
+    <a class="carousel-item" ><img src="https://i.imgur.com/UoVt0SD.jpg" alt="2019-image-1"/></a>
+    <a class="carousel-item" ><img src="https://i.imgur.com/tFJUdpY.jpg" alt="2019-image-2"/></a>
+    <a class="carousel-item" ><img src="https://i.imgur.com/EeFW9yt.jpg" alt="2019-image-3"/></a>
+    <a class="carousel-item" ><img src="https://i.imgur.com/ho1TpX3.jpg" alt="2019-image-4"/></a>
+    <a class="carousel-item" ><img src="https://i.imgur.com/NuVyjXK.jpg" alt="2019-image-5"/></a>
+    <a class="carousel-item" ><img src="https://i.imgur.com/r7ZMttJ.jpg" alt="2019-image-6"/></a>
+</div>
+
+<footer class="page-footer primaryColor">
+    <div class="container scrollspy" id="contact">
+        <h3 class="row center">Contact Us</h3>
+
+        <div class="row center contactBtns">
+            <a href="https://www.facebook.com/ieeeuoft" class="waves-effect waves-light" rel="noopener" target="_blank"><img src="{{ static('event/images/social/email.svg') }}" /></a>
+            <a href="https://www.ieee.utoronto.ca" class="waves-effect waves-light" rel="noopener" target="_blank"><img src="{{ static('event/images/social/website.svg') }}" /></a>
+            <a href="https://www.facebook.com/ieeeuoft" class="waves-effect waves-light" rel="noopener" target="_blank"><img src="{{ static('event/images/social/facebook.svg') }}" /></a>
+            <a href="https://www.instagram.com/ieee_uoft" class="waves-effect waves-light" rel="noopener" target="_blank"><img src="{{ static('event/images/social/instagram.svg') }}" /></a>
+        </div>
+    </div>
+    <div class="footer-copyright center">
+        <div class="container">
+            Copyright © 2020-2021 IEEE University of Toronto Student Branch
```

Comment:
Should we throw a link to the template repo in here too?

Reply:
What should I make it say